sklearn/metrics/_ranking.py

```diff
@@ -769,8 +778,10 @@ def precision_recall_curve(y_true, probas_pred, *, pos_label=None,
         True binary labels. If labels are not either {-1, 1} or {0, 1}, then
         pos_label should be explicitly given.
 
-    probas_pred : ndarray of shape (n_samples,)
-        Estimated probabilities or decision function.
+    y_score : ndarray of shape (n_samples,)
+        Target scores, can either be probability estimates of the positive
+        class, or non-thresholded measure of decisions (as returned by
+        `decision_function` on some classifiers).
```

Comment:
Maybe,
```suggestion
        class, or the output of the `decision_function`.
```
As "non-thresholded measure of decisions " can be confusing.

Reply:
I just made copy and paste from other functions... Should I also change it in the other functions then?